README.md

```diff
@@ -1,19 +1,18 @@
 # Concourse pool boy
 
-Like a an efficient pool boy, keep the pools of the [Concourse pool-resource] clean from debris.
-
-For an even smarter pool boy, you can tell him to check if a floating body is still alive before
-cleaning it by using our improved [Pix4D pool-resource] and setting the parameters
-`CONCOURSE_BASE_URL`, `CONCOURSE_USERNAME` and `CONCOURSE_PASSWORD` (see below).
+Observes the pools used by the *Concourse pool resource* and puts back to `unclaimed` any **stale** lock.
```

Comment:
[minor] What's the purpose of the two different emphasis? `unclaimed` vs **stale**

Reply:
It is because `unclaimed` is the directory where the file will be moved to, and **stale** is the concept. I agree it is not very nice. Let me do another attempt.

Reply:
I reworded it. Opinions?